### DISCUSSION TITLE
Ansible integration

### DISCUSSION THREAD

**Discussion by Cyben** [community/community#4207] [Codespaces] [answered]
Hey, 
The idea of Cloudsapce is appreciated and while registering to the waiting list, I haven’t seen there an ansible option as it is not a programming language.
But the option of auto completion of modules and a linter or auto formatter may be very useful for my opinion.

Would like to hear what do you think about that.

Thanks

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @Cyben ,thanks for your feedback. We do have a sample container template for Azure Ansible here: https://github.com/microsoft/vscode-dev-containers/tree/main/containers/azure-ansible

Hope this helps :)
